playbooks/spark-integration-test-minikube-k8s/run.yaml

```diff
@@ -3,7 +3,6 @@
   roles:
     - install-openjdk
     - role: install-docker
-      docker_version: 18.06
```

Comment:
19.03?

Reply:
I think latest is ok, now the latest release is 19.03, I don't know whether we have to hard it to 19.03.

Reply:
If so,  the job tags should be `Application:Docker@latest` as well, otherwise once docker release a new version, the tag will be wrong.

Reply:
OK, I think set the docker version is more clear, but now there are many jobs have the problem, maybe we should fix them.